src/index.js

```diff
@@ -87,6 +127,27 @@ ${stories.map(({ name }) => `
       config.resolve.alias['@storybook/react'] = path.resolve(__dirname, './shim')
       config.resolve.alias['docz-plugin-storybook/story-wrapper'] = require.resolve(storyWrapper)
 
+      const mdxRule = config.module.rules
+        .find((rule) => rule.use && rule.use[0] && rule.use[0].loader && rule.use[0].loader.endsWith('?id=mdx'))
+
+      if (mdxRule) {
```

Comment:
(Silvi on Travis' laptop because flights and shit)

Are we planning on removing this in the future? If docz supports `markdown` `md` and `mdx` it seems like this might eventually cause some conflicts with consumers.

Reply:
I think the best way to remove it in the future would be aliasing the `storybook-readme` addon.